README.md

```diff
@@ -1,6 +1,10 @@
 ## Calico Network Policy for Kubernetes 
 
-This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod. 
+This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  
+
+![calico-policy-controller](calico-policy-controller.png)
+
+The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod. 
```

Comment:
I know this was there already, but the policy controller isn't actually a set of replicated pods - it's a single pod.